build.sbt

```diff
@@ -1,6 +1,9 @@
 import Settings._
 import Testing._
 
+ThisBuild / libraryDependencySchemes += "org.typelevel" %% "cats-effect" % "always"
```

Comment:
Why do we need this here rather than in workbench libs as one of the dependencies there? Alternatively, what does this do? I just notice that there's no other uses of this sort of approach in this file, so am curious!

Reply:
yea these shouldn't be needed